include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -696,6 +696,8 @@ typedef void (*OrtCustomJoinThreadFn)(OrtCustomThreadHandle ort_custom_thread_ha
 
 typedef OrtStatus*(ORT_API_CALL* RegisterCustomOpsFn)(OrtSessionOptions* options, const OrtApiBase* api);
 
+typedef void (*RunAsyncCallbackFn)(OrtValue*, size_t, OrtStatusPtr);
```

Comment:
Why not have OrtStatusPtr as a return type instead of void?

Reply:
This is the callback - ort call this function to notify customer the result, and status.